app/build.gradle

```
@@ -4,13 +4,13 @@ plugins {
 }
 
 android {
-    compileSdk 33
+    compileSdk = 36
```

Please be consistent across the board. You added "=" to some of the parameters (like you need to do when using Kotlin DSL), but not to all of them. For simplicity and consistency reasons, please keep the same structure (which is fine for Gradle DSL) and just update the versions.

```suggestion
    compileSdk 36
```

---

Here, if I don't use '=', compileSdk is marked as deprecated. Is it ok to leave it as deprecated?

---

Alright, yes, it's fine.